src/javascripts/modules/CollapseAccordion.js

```diff
@@ -0,0 +1,18 @@
+export default class CollapseAccordion {
+  constructor(el) {
+    this.el = el;
+
+    this.el.addEventListener("click", e => {
+      e.preventDefault();
+      this.toggleClass(this.el.parentElement);
+    });
+  }
+
+  toggleClass(t) {
+    t.classList.toggle("is-collapsed");
+    const sprites = t.children[0].children;
+    for (const sprite of sprites) {
+      sprite.classList.toggle("u-hide");
+    }
+  }
+}
```

Comment:
Looking at your html

```
<div class="c-accordion__wrapper">
    <p class="c-accordion__header" aria-expanded="false" aria-controls="accordion-3" data-module="CollapseAccordion">
        Footer Section 3 
        {{ sprite("plus", "0 0 1 1", "c-icon c-icon--tiny c-icon--darkbg c-accordion__icon") }}
        {{ sprite("chev", "0 0 1 1", "c-icon c-icon--tiny c-icon--darkbg c-accordion__icon  u-hide") }}
    </p>
    <ul class="o-list-bare  c-accordion__list" id="accordion-3" aria-hidden="true">
        <li class="o-list-bare__item c-accordion__panel_text"><a></a></li>
        <li class="o-list-bare__item c-accordion__panel_text"><a></a></li>
        <li class="o-list-bare__item c-accordion__panel_text"><a></a></li>
    </ul>
</div>
```

Why would you `this.el.parentElement` and then `t.children[0]` ?

You could refactor your code to look like this

```
this.el.addEventListener("click", e => {
    e.preventDefault();
    this.toggleClass(this.el);
});

toggleClass(t) {
    t.parentElement.toggle("is-collapsed"); // selects .c-accordion__wrapper
    
    const sprites = Array.from(t.querySelectorAll("sprite")); // Array.from makes it into an Iterable item
    for (const sprite of sprites) {
        sprite.classList.toggle("u-hide");
    }
}
```

Also making it into a function doesn't have much use here but it's fine